sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -84,6 +86,7 @@
           .put(TypeName.DECIMAL, BigDecimal::new)
           .put(TypeName.BOOLEAN, Boolean::valueOf)
           .put(TypeName.STRING, str -> str)
+          .put(TypeName.DATETIME, str -> new DateTime(((long)Double.parseDouble(str)) * 1000))
```

Comment:
The failure is in `:beam-sdks-java-io-google-cloud-platform:checkstyleMain` because no space after the `(long)` cast.

Reply:
Yes, it seems like `./gradlew :beam:sdks:java:extensions:sql:check` does not trigger other dependencies' style check.